### ISSUE TITLE
Bug in example: turn_based

### ISSUE THREAD

**Issue by CitrusWire** [MatejSloboda/Dijkstra_map_for_Godot#99]
The turn_based example scene doesn't run with F6.
There's a parser error. You're using `id` as a variable in the for loop when it has already been declared earlier in the function (`redraw_movement_access`).

3.2.3

**Comment by dianegolfouse** [comment]
I see it was fixed on master :slightly_smiling_face: With the recent changes in the project's structure, maybe it is time we release a new version to the asset store.
I think @MatejSloboda has the rights for this ?

**Comment by CitrusWire** [comment]
There's a bug in the project_example too. I had it crash out when I was clicking around with the dragon:

![image](https://user-images.githubusercontent.com/72134064/104357549-41149f00-5505-11eb-9ff4-c5371c486d6e.png)

I've tried replicating with a few hundred clicks but it seems to have been one of those annoying rare bugs. :-(
Figured I'd raise it here so you're aware of it. Feel free to create a new issue if you want.
